app/contributing/community.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Community
+no_version: true
+---
+
+This section contains information on how to contribute using our docs-as-code approach. You may also be interested in [Community Expectations](/contributing/community-expectations/) and [Hackathons](/contributing/hackathons/). 
+
+## How to contribute to docs-as-code as a beginner
+
+Welcome to our introduction on how to contribute to Kong Docs using our docs-as-code approach. This tutorial is designed for first-time contributors who want to get started with simple tasks to boost their confidence in tooling, add to their portfolio, and join our international community. 
+
+We house all our docs code on GitHub, a code versioning platform that allows people all over the world to collaborate on projects. You'll need a [GitHub](https://github.com/) account, so if you don't already have one, now's the time to sign up. 
+
+Once you have a GitHub account, head over to our [docs.konghq.com](https://github.com/Kong/docs.konghq.com) project. 
+
+The first thing we’re going to do is look over the README. Read through the intro section for some basic information about our project. 
+
+### Basic setup
+
+To contribute using our docs-as-code approach, you'll need:
+
+* A code editor like [VScode](https://code.visualstudio.com/) or [Atom](https://atom.io/).
+* Basic Git knowledge, see [Git/GitHub Resources](/contributing/#gitgithub-resources).
+* A local copy of our repository. On [our main repository page](https://github.com/Kong/docs.konghq.com), click the `Code` dropdown button and copy the `ssh` URL. In your terminal, run `git clone {SSH_URL}`, replacing `{SSH_URL}` with the URL you copied from the `Code` dropdown button. 
+* To know the difference between a PR and an Issue. PR stands for Pull Request or Please Review. It’s different from an Issue in that an Issue is a statement of something that needs attention, while a PR attempts to resolve something. So, an Issue is an idea and a PR is an action. 
+
+From here, determine which applies to you:
+
+* [I know what I want to contribute](#i-know-what-i-want-to-contribute).
+* [I need ideas on what to contribute](#i-need-ideas-on-what-to-contribute).
+
+### I know what I want to contribute
+
+Great! Let's first check to see if someone had the same idea as you. Go to [Issues](https://github.com/Kong/docs.konghq.com/issues) and read through the titles to see if something sounds similar to what you want to contribute or fix. 
+
+If you find an Issue that aligns with what you want to contribute, go ahead and assign it to yourself in the right-side column and add a comment stating that you're going to be working on this task. Then, head to [Make your contribution](#make-your-contribution). 
```

Comment:
I think this needs a line on what happens if you know what you want to do but there are no open issues for it.

Reply:
Sure, I think the contributor will not need to open an Issue, and they can instead just start working locally and push their PR. Is this in alignment with your thoughts?

Reply:
Agree - that makes sense to me. If they have an idea with no related open issue, they can absolutely go ahead and start working on it directly.